### DISCUSSION TITLE
rio-core dev

### DISCUSSION THREAD

**Discussion by kzali** [multigcs/riocore#17] [Q&A] [answered]
Hi Oliver
I had read in one of the discussions that you have merged the gui  repository to rio-core.  Do we still use the following method for main or dev?
git clone https://github.com/multigcs/riocore.git
git clone https://github.com/multigcs/riogui.git
cd riocore
sorry this might be a silly question

Also is the eth01 the same code as RIO?

**Comment by multigcs** [comment]
riogui repository is now deleted, the bin/rio-setup, ... files now in the riocore repository.
so you need only 'https://github.com/multigcs/riocore.git' in the feature

git clone https://github.com/multigcs/riocore.git
cd riocore

PYTHONPATH=. bin/rio-setup riocore/configs/Tangoboard/config.json

**Reply by kzali** [reply]
Thanks 
So the ETH01 UDP is not supported in riocore

**Reply by multigcs** [reply]
ETH01 UDP ???

**Reply by multigcs** [reply, marked as answer]
ahh, sorry, the wt32-eth0 ..

This will continue to be supported,
only the sources are still in the old LinuxCNC-RIO repository: 

https://github.com/multigcs/LinuxCNC-RIO/tree/main/UDP2SPI-Bridge

**Reply by kzali** [reply]
Great thanks
great work
